Testing/ORKTest/ORKTest/TaskFactory.h

```diff
@@ -0,0 +1,93 @@
+//
+//  TaskFactory.h
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
```

Comment:
Thanks for spotting, fixed!

I took the chance to remove the *acute accent* (which I no longer use) from my name in all of the rest of files, I hope you don't mind.